lib/mayaUsd/render/vp2ShaderFragments/shaderFragments.cpp

```diff
@@ -248,12 +248,16 @@ MStatus HdVP2ShaderFragments::registerFragments()
 // Fragment deregistration
 MStatus HdVP2ShaderFragments::deregisterFragments()
 {
+    // If it was never registered, leave as-is:
+    if (!_registrationCount) {
```

Comment:
We have a few places with a similar pattern. Can you make sure we have this fix in there as well?